models/surgical_tool_segmentation/docs/readme.md

```diff
@@ -0,0 +1,95 @@
+The training was performed with an at least 12GB-memory GPU.
+
+Actual Model Input: 736 x 480 x 3
+
+## Input and output formats
+Input: 3 channel video frames
+
+Output: 2 channels: Label 1: tools; Label 0: everything else
+
+## Scores
+This model achieves the following IoU score on the test dataset (our own split from the first batch data):
+
+Mean IoU = 0.87
+
+## commands example
+Execute training:
+
+```
+python -m monai.bundle run training --meta_file configs/metadata.json --config_file configs/train.json --logging_file configs/logging.conf
+```
+
+Override the `train` config to execute evaluation with the trained model:
+
+```
+python -m monai.bundle run evaluating --meta_file configs/metadata.json --config_file "['configs/train.json','configs/evaluate.json']" --logging_file configs/logging.conf
+```
+
+Execute inference:
+
+```
+python -m monai.bundle run evaluating --meta_file configs/metadata.json --config_file configs/inference.json --logging_file configs/logging.conf
+```
+
+Export checkpoint to TorchScript file:
+
+```
+python -m monai.bundle ckpt_export network_def --filepath models/model.ts --ckpt_file models/model.pt --meta_file configs/metadata.json --config_file configs/inference.json
+```
+
+Export checkpoint to onnx file, which has been tested on pytorch 1.12.0:
+
+```
+python scripts/export_to_onnx.py --model models/model.pt --outpath models/model.onnx
+```
+
+Export TorchScript file to a torchscript module targeting a TensorRT engine with float16 precision.
+
+```
+torchtrtc -p f16 models/model.ts models/model_trt.ts "[(1,3,736,480);(4,3,736,480);(8,3,736,480)]"
+```
+The last parameter is the dynamic input shape in which each parameter means "[(MIN_BATCH, MIN_CHANNEL, MIN_WIDTH, MIN_HEIGHT), (OPT_BATCH, .., ..., OPT_HEIGHT), (MAX_BATCH, .., ..., MAX_HEIGHT)]". Please notice if using docker, the TensorRT CUDA must match the environment CUDA and the Torch-TensorRT c++&python version must be installed. For more examples on how to use the Torch-TensorRT, you can go to this [link](https://pytorch.org/TensorRT/). The [github source code link](https://github.com/pytorch/TensorRT) here shows the detail about how to install it on your own environment.
+
+Export TensorRT float16 model from the onnx model:
+
+```
+trtexec --onnx=models/model.onnx --saveEngine=models/model.trt --fp16 --minShapes=INPUT__0:1x3x736x480 --optShapes=INPUT__0:4x3x736x480 --maxShapes=INPUT__0:8x3x736x480 --shapes=INPUT__0:4x3x736x480
+```
+This command need TensorRT with correct CUDA installed in the environment. For the detail of installing TensorRT, please refer to [this link](https://docs.nvidia.com/deeplearning/tensorrt/install-guide/index.html). In addition, there are padding operations in this FlexibleUNet structure that not support by TensorRT. Therefore, when tried to convert the onnx model to a TensorRT engine, an extra step shown below is needed to execute. The "new_model.onnx" should be the "model.onnx" used to do converting.
```

Comment:
Suggest to move this polygraphy step before the previous trtexec step and change the onnx model name in the trtexec command.

Reply:
Hi @jin-nvidia ,
     Thank you for your advice. Polygraphy step was moved.
Thanks,
Bin